test/unit/data/ui_test.js

```diff
@@ -78,5 +78,21 @@ describe('ui', () => {
 
       dataScenarioConfig.scenarios.forEach(scenario => should.equal(helper, scenario.test.config[helperName]));
     });
+
+    it("should shows object's toString() method in each scenario's name if the toString() method is overrided", () => {
```

Comment:
I think this test case name is too redundant...
If you have any idea, please give me an advice for me ;)

Reply:
Good name!